app/Website/src/Http/Controllers/NewsletterSubscribeController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Website\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use Knuckles\Scribe\Attributes\Endpoint;
+use Spatie\MailcoachSdk\Facades\Mailcoach;
+use App\Website\Http\Requests\SubscribeRequest;
+
+class NewsletterSubscribeController extends Controller
+{
+    #[Endpoint('Website/Subscribe', <<<'DESC'
+  This endpoint is for subscribing to Leanhub.mk.
+  Additionally it uses Cloudflare Turnstile ReCaptcha for validation.
+ DESC)]
+    public function store(SubscribeRequest $request)
+    {
+
+        $first_name = $request->input('first_name');
+        $email = $request->input('email');
+
+        $subscriber = Mailcoach::createSubscriber(
+            emailListUuid: config('mailcoach-sdk.email_list'),
+            attributes: [
+                'first_name' => $first_name,
+                'email' => $email,
+        ]
```

Comment:
Push the bracket inside to be inline with attributes, you can use tab for that.

Reply:
Done